Sources/MapboxNavigation/WayNameView.swift

```diff
@@ -80,9 +80,11 @@ open class WayNameView: UIView {
         }
         
         if let roadName = feature.properties["name"] as? String {
```

Comment:
Maybe one `if` with two conditions, instead of two nested `if`s?

Reply:
Thanks for the suggestion, I'll improve this part.